cmd/data-usage_test.go

```diff
@@ -364,18 +364,19 @@ func TestDataUsageUpdatePrefix(t *testing.T) {
 			}
 			if e == nil {
 				t.Fatal("got nil result")
-			}
-			if e.Size != int64(w.size) {
-				t.Error("got size", e.Size, "want", w.size)
-			}
-			if e.Objects != uint64(w.objs) {
-				t.Error("got objects", e.Objects, "want", w.objs)
-			}
-			if e.Versions != uint64(w.objs) {
-				t.Error("got versions", e.Versions, "want", w.objs)
-			}
-			if e.ObjSizes != w.oSizes {
-				t.Error("got histogram", e.ObjSizes, "want", w.oSizes)
+			} else {
+				if e.Size != int64(w.size) {
```

Comment:
What a horrible change. So it isn't picking up that Fatal will exit? Let's do..
```suggestion
				return
			}
			if e.Size != int64(w.size) {
```
...

Reply:
I know